gatling-engine/src/main/resources/data/imsis.csv

```diff
@@ -1,4 +1,4 @@
-imsi,originHost, originRealm,destinationRealm
-999995000000000,dea01.node.epc.mnc001.mcc001.3gppnetwork.org,epc.mnc001.mcc001.3gppnetwork.org,epc.mnc001.mcc001.3gppnetwork.org
-999995000000001,dea01.node.epc.mnc001.mcc001.3gppnetwork.org,epc.mnc001.mcc001.3gppnetwork.org,epc.mnc001.mcc001.3gppnetwork.org
-999995000000002,dea01.node.epc.mnc001.mcc001.3gppnetwork.org,epc.mnc001.mcc001.3gppnetwork.org,epc.mnc001.mcc001.3gppnetwork.org
+imsi, airNumberOfVectors
+999995000000000, 1
+999995000000001, 2
+999995000000002, 3
```

Comment:
i'd suggest to create a feeder here. much nicer 
https://gatling.io/docs/current/session/feeder/